webui/react/src/pages/F_ExpList/F_ExperimentList.tsx

```diff
@@ -159,17 +159,49 @@ const F_ExperimentList: React.FC<Props> = ({ project }) => {
     }
   }, [settings.filterset, isLoadingSettings]);
 
-  const [selectedExperimentIds, setSelectedExperimentIds] = useState<number[]>([]);
-  const [excludedExperimentIds, setExcludedExperimentIds] = useState<Set<number>>(
-    new Set<number>(),
+  const selectAllIfLoaded = useMemo(
```

Comment:
nit: is the addition of `IfLoaded` to these variable names meaningful?

Reply:
I had named `columnsIfLoaded` in a previous PR to be the default value until settings finish loading. Since it is the value and not Loadable<value>  it may be clearer to drop the IfLoaded?

Reply:
dropped `IfLoaded` from these variables in a recent commit